test/scenarios/classic/install.go

```diff
@@ -51,8 +51,8 @@ func isAgentInjected(sampleApp sampleapps.SampleApp) features.Func {
 		for _, podItem := range pods.Items {
 			require.NotNil(t, podItem)
 
-			executionQuery := pod.NewExecutionQuery(podItem, sampleApp.ContainerName(), shell.ListDirectory("/var/lib/dynatrace")...)
-			executionResult, err := executionQuery.Execute(environmentConfig.Client().RESTConfig())
+			listCommand := shell.ListDirectory("/var/lib/dynatrace")
+			executionResult, err := pod.NewExecutionQuery(ctx, resources, podItem, sampleApp.ContainerName(), listCommand...).Execute()
```

Comment:
One could argue, that the codebase could benefit from a common style in for writing this repeatedly used line of code. I actually like this incarnation more than the two-statement versions above.